### ISSUE TITLE
SyntaxError on deno info 

### ISSUE THREAD

**Issue by dangry** [denoland/deno#6931]
```
➜ deno --version
deno 1.2.2
v8 8.5.216
typescript 3.9.2
```

Running `deno info --no-check cli.ts` on https://github.com/halvardssm/deno-nessie throws the following error:

```
error: Uncaught SyntaxError: Unexpected token '['
  public available_actions: Array<Command> = [];
^
    at <https://deno.land/x/denomander@0.6.2/src/Kernel.ts>:35:1
```

**Comment by ebebbington** [comment]
FYI kind of unrelated, but you won't be able to run it on deno v1.2.2 because dependencies nessie uses are std v0.55.0

**Comment by kitsonk** [comment]
@bartlomieju looks like the type stripping isn't stripping types. 😭

**Comment by Yamboy1** [comment]
@ebebbington this was while trying to get a graph of dependencies to help fix that :wink:

**Comment by ebebbington** [comment]
> @ebebbington this was while trying to get a graph of dependencies to help fix that 😉

Oh yeah course! 🤣

**Comment by bartlomieju** [comment]
https://github.com/swc-project/swc/issues/921

**Comment by bartlomieju** [comment]
This issue will be fixed in `v1.2.3` next week.
